docs/tutorial-basics/create-a-page.md

```diff
@@ -31,8 +31,9 @@ Add a Route for the New Page: Below the existing routes, add a route for your ne
 ```ts
 // Home Route
 const MyNewPageRoute: PageType = {
-  index: true,
+  index: false,
```

Comment:
We can completely omit index parameter in all routes except the Home route which doesn't have path. **PageType** is union of my custom **MenuType** type and **React Router** package [RouteObject](https://reactrouter.com/en/main/route/route). But, I'll merge this PR and then update this section with more info later.